webapp/locales/en.json

```diff
@@ -306,9 +306,9 @@
   "editor": {
     "embed": {
       "always_allow": "Always allow embedded content by third party providers (this setting can be changed any time)",
-      "data_privacy_info": "Your data has not yet been shared with any third party providers. If you proceed to watch this video the following provider will likely collect user data:",
+      "data_privacy_info": "Your data has not yet been shared with any third party providers. If you proceed to play this content the following provider will likely collect user data:",
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-08-05T14:33:17Z" title="Wednesday, August 5th 2020, 4:33:17 pm +02:00">Aug 5, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -306,9 +306,9 @@
   "editor": {
     "embed": {
       "always_allow": "Always allow embedded content by third party providers (this setting can be changed any time)",
-      "data_privacy_info": "Your data has not yet been shared with any third party providers. If you proceed to watch this video the following provider will likely collect user data:",
+      "data_privacy_info": "Your data has not yet been shared with any third party providers. If you proceed to watch this content the following provider will likely collect user data:",
```


```suggestion
      "data_privacy_info": "Your data has not yet been shared with any third party providers. If you proceed to play this content the following provider will likely collect user data:",
```

We've overseen to replace the word `watch` with `play` @Thiruppathi , because it shall be working for sound only as well.

Reply:
> [<img alt="Thiruppathi" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Thiruppathi) **Authored by [Thiruppathi](https://github.com/Thiruppathi)**
_<time datetime="2020-08-05T14:57:15Z" title="Wednesday, August 5th 2020, 4:57:15 pm +02:00">Aug 5, 2020</time>_

---


Haa.. good point. Will commit this suggestion.